sycl/include/sycl/ext/intel/esimd/detail/simd_obj_impl.hpp

```diff
@@ -354,6 +354,11 @@ class simd_obj_impl {
 #endif
   }
 
+  /// @return A reference to the value of the
+  /// underlying raw vector. Intended for use
+  /// with inline assembly.
+  raw_vector_type &data_ref() { return M_data; }
```

Comment:
Just idea:
Is it possible to reuse the name 'data()', similarly to std::vector::data(): https://en.cppreference.com/w/cpp/container/vector/data
or it is not doable, because the returned value of data() and data_ref() differs not only by a 'const' specifier?

Reply:
no we can do that, good idea, thanks